speech/detector.go

```diff
@@ -269,6 +269,10 @@ func (sd *Detector) Reset() error {
 	return nil
 }
 
+func (sd *Detector) SetThreshold(value float32) {
+	sd.cfg.Threshold = value
+}
```

Comment:
@kisasexypantera94 It makes sense, but depending on how this is used, it could be racy. I assume you are setting this between one detection and the next?

Reply:
Yes. It might make sense to make this variable atomic, but I don’t have such a need yet, and also didn’t want to break compatibility